### PR TITLE
Refactor istio-clean-iptables

### PR DESCRIPTION
**Pull request by abhide**
* Use cobra/viper for command line args

### CODE REVIEW

File: tools/istio-clean-iptables/pkg/cmd/cleanup.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	dep "istio.io/istio/tools/istio-iptables/pkg/dependencies"
+)
+
+func removeOldChains(ext dep.Dependencies, cmd string) {
+	// Remove the old chains
+	ext.RunQuietlyAndIgnore(cmd, "-t", constants.NAT, "-D", constants.PREROUTING, "-p", constants.TCP, "-j", constants.ISTIOINBOUND)
+	ext.RunQuietlyAndIgnore(cmd, "-t", constants.MANGLE, "-D", constants.PREROUTING, "-p", constants.TCP, "-j", constants.ISTIOINBOUND)
+	ext.RunQuietlyAndIgnore(cmd, "-t", constants.NAT, "-D", constants.OUTPUT, "-p", constants.TCP, "-j", constants.ISTIOOUTPUT)
+	// Flush and delete the istio chains.
+	ext.RunQuietlyAndIgnore(cmd, "-t", constants.NAT, "-F", constants.ISTIOOUTPUT)
```
Comment (rlenglet) [inline tools/istio-clean-iptables/pkg/cmd/cleanup.go]: While you're touching this code, it would be cleaner to iterate over a list of chain names.

Comment (abhide) [reply tools/istio-clean-iptables/pkg/cmd/cleanup.go]: Done.

**Comment by abhide** [comment]
@istio/wg-test-and-release-maintainers Could one of you take a look?

**Pull request merged**